packages/amazonq/test/e2e/amazonq/VET.test.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Workbench, By, EditorView, WebviewView } from 'vscode-extension-tester'
+
+describe('Amazon Q Login Flow', function () {
+    // need this timeout because Amazon Q takes awhile to load
+    this.timeout(90000)
+    let webviewView: WebviewView
+
+    // this before() block will run before every test
+    // NOTE: I tested all the timeouts and they are necessary for the webview to load properly
+    before(async function () {
```

Comment:
I think in mocha `before` runs once in the beginning and then `beforeEach` runs before each test.